src/main/resources/config/liquibase/master.xml

```diff
@@ -19,5 +19,8 @@
     <include file="classpath:config/liquibase/changelog/20190410101919_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20190416170046_changelog.xml" relativeToChangelogFile="false"/>
     <include file="classpath:config/liquibase/changelog/20190520032531_changelog.xml" relativeToChangelogFile="false"/>
+    <include file="classpath:config/liquibase/changelog/20190526152122_added_entity_ProgrammingExerciseTestCase.xml" relativeToChangelogFile="false"/>
```

Comment:
please merge all change sets into one file and one <changeSet>

Reply:
So I merged both changesets into one file.
However if I do one of the following, liquibase complains that the table alread exists (even though I ran clear checksums):
- Rename the xml file name
- Rename the changeset id

Reply:
Solved :)